core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -684,4 +684,13 @@ private RequestId nextRequestId() {
             return id;
         }
     }
+
+    protected void performConfigCheck(ServerConfig config) {
+        // This is temporary hack to switch original config with new config for older HttpServerHandlers.
+        // TODO - Figure out a way to close the open channel so these handlers get garbage collected and
+        // not serve any traffic.
```

Comment:
Also need to figure out a way to remove the subscribers that are stale and no longer needed to be kept around.

Reply:
The above comment is no longer relevant.